src/uwtools/tests/test_rocoto.py

```diff
@@ -52,3 +53,17 @@ def test_write_rocoto_xml(tmp_path):
 
     expected = support.fixture_path("hello_workflow.xml")
     support.compare_files(expected, output)
+
+
+def test_rocoto_xml_is_valid():
+    with resources.as_file(resources.files("uwtools.resources")) as resc:
+        with open(resc / "schema_with_metatasks.rng", "r", encoding="utf-8") as f:
+            schema = etree.RelaxNG(etree.parse(f))
+
+    valid_xml = support.fixture_path("hello_workflow.xml")
+    tree = etree.parse(valid_xml)
+    assert schema.validate(tree)
+
+    invalid_xml = support.fixture_path("rocoto_invalid.xml")
+    tree = etree.parse(invalid_xml)
+    assert not schema.validate(tree)
```

Comment:
linter complained at `assert False == schema.validate(tree)`. Let me know if this is the way we want to go here.

Reply:
Try `assert schema.validate(tree) is False` and see if the linter is happy with that. Normally `pylint` complains about `== None`, `== True`, and `== False` and prefers `is` to `==` in these cases (maybe some others, I'm not sure). The rationale is that those values are _singleton_ objects in Python, and `is` is a stronger test than `==`; `is` means that the two things are really the exact same object in memory, which must be the case with singletons, while `==` means that two values are somehow equivalent, however equivalence is defined for those values. There's nothing really wrong with `==` that I'm aware of, but using `is` is more idiomatic.

Using `not` can be dicey in Python. I mean, definitely do not avoid using it, but understand and consider that a lot of values can be "falsey" in Python, including an empty `list`, an empty `dict`, and empty `str`, the number 0, `False`, and `None`. Probably I'm forgetting some! For example, if you know you've got some kind of collection `c` -- a `list`, a `dict`, a `set`, etc. -- saying `not c` is the idiomatic way to indicate an empty collection, much preferred to `len(c) == 0`. But when using `not`, think about what kind of values it's being applied to and whether there's any chance of them being falsey when that's not what you intend. For example, if a function argument was typed `x: Optional[list] = None`, using `not x` to try to detect when `x` wasn't passed in by the caller (i.e. has a `None` value) isn't a great idea, since a user might have passed `[]` in as a value, which is also falsey.

Reply:
@christinaholtNOAA based on your comment elsewhere on the PR

>> Does it make sense to simplify this line with:
>> `assert not schema.validate(tree)`